chainer/links/caffe/caffe_function.py

```diff
@@ -371,14 +371,14 @@ def _setup_scale(self, layer):
         # Case of only one bottom where W is learnt parameter.
         if len(bottom) == 1:
             W_shape = blobs[0].shape.dim
-            func = _Scale(axis, W_shape, bias_term)
+            func = links.Scale(axis, W_shape, bias_term)
```

Comment:
Please do not use alias defined in `chainer.links.__init__` inside `Chainer` module. Use `links.scale.Scale` instead.